README.md

```diff
@@ -1,446 +1,35 @@
 
-[![npm version](https://badge.fury.io/js/express-graphql.svg)](https://badge.fury.io/js/express-graphql)
-[![Build Status](https://github.com/graphql/express-graphql/workflows/CI/badge.svg?branch=master)](https://github.com/graphql/express-graphql/actions?query=branch%3Amaster)
-[![Coverage Status](https://codecov.io/gh/graphql/express-graphql/branch/master/graph/badge.svg)](https://codecov.io/gh/graphql/express-graphql)
+# This library is deprecated
 
-Create a GraphQL HTTP server with any HTTP web framework that supports connect styled middleware, including [Connect](https://github.com/senchalabs/connect) itself, [Express](https://expressjs.com) and [Restify](http://restify.com/).
+`express-graphql` was the first official reference implementation of using GraphQL with HTTP. It has existed since 2015 and was mostly unmaintained in recent years.
 
-## Installation
+The official [GraphQL over HTTP](https://github.com/graphql/graphql-over-http) work group is standardizing the way you transport GraphQL over HTTP and it made great progress bringing up the need for a fresh reference implementation.
 
-```sh
-npm install --save express-graphql
-```
+Please read the [GraphQL over HTTP spec](https://graphql.github.io/graphql-over-http) for detailed implementation information.
 
-### TypeScript
+## Say hello to [`graphql-http`](https://github.com/enisdenjo/graphql-http)
 
-This module includes a [TypeScript](https://www.typescriptlang.org/)
-declaration file to enable auto complete in compatible editors and type
-information for TypeScript projects.
+[`graphql-http`](https://github.com/enisdenjo/graphql-http) is now the GraphQL official reference implementation of the [GraphQL over HTTP spec](https://graphql.github.io/graphql-over-http).
 
-## Simple Setup
+## For users
 
-Just mount `express-graphql` as a route handler:
+As a reference implementation, [`graphql-http`](https://github.com/enisdenjo/graphql-http) implements exclusively the [GraphQL over HTTP spec](https://graphql.github.io/graphql-over-http/).
 
-```js
-const express = require('express');
-const { graphqlHTTP } = require('express-graphql');
+In case you're seeking for a full-featured experience (with file uploads, @defer/@stream directives, subscriptions, etc.), you're recommended to use some of the great JavaScript GraphQL server options:
 
-const app = express();
+- [`graphql-yoga`](https://www.the-guild.dev/graphql/yoga-server) ([fully compliant](https://github.com/enisdenjo/graphql-http/tree/master/implementations/graphql-yoga), [migration guide](https://www.the-guild.dev/graphql/yoga-server/v3/migration/migration-from-express-graphql))
+- [`apollo-server`](https://www.apollographql.com/docs/apollo-server/) ([partially compliant](https://github.com/enisdenjo/graphql-http/tree/master/implementations/apollo-server))
+- [`mercurius`](https://mercurius.dev/) ([partially compliant](https://github.com/enisdenjo/graphql-http/tree/master/implementations/mercurius))
```

Comment:
"Partially compliant" seems uncharitable; they're all fully compliant in that they implement all the MUSTs. Yoga is fully compliant _including_ implementing optional features such as `GET` and the various `SHOULD`s, and is watershed-ready.

Also, we should invite people to link their own servers here.

```suggestion
- [`graphql-yoga`](https://www.the-guild.dev/graphql/yoga-server) ([fully compliant (0 warnings)](https://github.com/enisdenjo/graphql-http/tree/master/implementations/graphql-yoga), [migration guide](https://www.the-guild.dev/graphql/yoga-server/v3/migration/migration-from-express-graphql))
- [`apollo-server`](https://www.apollographql.com/docs/apollo-server/) ([compliant](https://github.com/enisdenjo/graphql-http/tree/master/implementations/apollo-server))
- [`mercurius`](https://mercurius.dev/) ([compliant](https://github.com/enisdenjo/graphql-http/tree/master/implementations/mercurius))

To add your server to this list, please first add an implementation test to https://github.com/enisdenjo/graphql-http/tree/master/implementations/
```

@enisdenjo I'm not sure if this is the link you want to send people to, just a suggestion